pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -1,4 +1,5 @@
 %link{:href=>"/shared/css/warning-banner.css", :rel=>"stylesheet"}
+%script{src: minifiable_asset_path("js/pegasusCommon.js")}
```

Comment:
We have a few places where this gets used, so we'd start making lots of entry points which doesn't seem ideal. (code.org and hourofcode.com don't have single top level haml files). 
@pcardune do you have strong opinions on this?